modules/service/src/main/scala/lucuma/odb/service/ConfigurationService.scala

```diff
@@ -162,13 +162,16 @@ object ConfigurationService {
       pids: List[Program.Id], 
       oids: List[Observation.Id]
     ): ResultT[F, (Map[Program.Id, List[ConfigurationRequest]], Map[(Program.Id, Observation.Id), Configuration])] =
-      ResultT:
-        services.runGraphQLQuery(Queries.RequestsAndConfigurations(pids, oids)).map: r =>
-          r.flatMap: json =>            
-            import Queries.RequestsAndConfigurations.given
-            json.hcursor.as[Queries.RequestsAndConfigurations.Response] match
-              case Right(r)    => Result(r)
-              case Left(other) => Result.internalError(other.getMessage)
+      if pids.isEmpty && oids.isEmpty then
+        ResultT.pure((Map.empty[Program.Id, List[ConfigurationRequest]], Map.empty[(Program.Id, Observation.Id), Configuration]))
```

Comment:
When testing with the full `dev` database contents, the obscalc workers would always hang here when `pids` and `oids` were empty.  I'm not sure why that is, but it seems like it is valid to skip the call when we know the result will be empty anyway?